docs/conf.py

```diff
@@ -88,6 +89,7 @@
 #
 html_theme = 'sphinx_book_theme'
 # html_style = 'css/flax_theme.css'
+html_css_files = ["css/flax_theme.css"]
```

Comment:
So it was actually not being used?
Also, remove the line above?

Reply:
Seems that it was not being used with the new theme.